storage/entities/peer.js

```diff
@@ -0,0 +1,304 @@
+ * @property {number} [wsPort_eql]
+ * @property {number} [wsPort_ne]
+ * @property {number} [wsPort_gt]
+ * @property {number} [wsPort_gte]
+ * @property {number} [wsPort_lt]
+ * @property {number} [wsPort_lte]
+ * @property {number} [wsPort_in]
+ * @property {number} [state]
+ * @property {number} [state_eql]
+ * @property {number} [state_ne]
+ * @property {number} [state_gt]
+ * @property {number} [state_gte]
+ * @property {number} [state_lt]
+ * @property {number} [state_lte]
+ * @property {number} [state_in]
+ * @property {string} [os]
+ * @property {string} [os_eql]
+ * @property {string} [os_ne]
+ * @property {string} [os_in]
+ * @property {string} [os_like]
+ * @property {string} [version]
+ * @property {string} [version_eql]
+ * @property {string} [version_ne]
+ * @property {string} [version_in]
+ * @property {string} [version_like]
+ * @property {number} [clock]
+ * @property {number} [clock_eql]
+ * @property {number} [clock_ne]
+ * @property {number} [clock_gt]
+ * @property {number} [clock_gte]
+ * @property {number} [clock_lt]
+ * @property {number} [clock_lte]
+ * @property {number} [clock_in]
+ * @property {string} [broadhash]
+ * @property {string} [broadhash_eql]
+ * @property {string} [broadhash_ne]
+ * @property {string} [broadhash_in]
+ * @property {string} [broadhash_like]
+ * @property {number} [height]
+ * @property {number} [height_eql]
+ * @property {number} [height_ne]
+ * @property {number} [height_gt]
+ * @property {number} [height_gte]
+ * @property {number} [height_lt]
+ * @property {number} [height_lte]
+ * @property {number} [height_in]
+ */
+
+class Peer extends BaseEntity {
+	/**
+	 * Constructor
+	 * @param {BaseAdapter} adapter - Adapter to retrieve the data from
+	 * @param {filters.Peer} defaultFilters - Set of default filters applied on every query
+	 */
+	constructor(adapter, defaultFilters = {}) {
+		super(adapter, defaultFilters);
+
+		this.addField('id', 'number', { filter: filterType.NUMBER });
+		this.addField('ip', 'string', { format: 'ip', filter: filterType.TEXT });
+		this.addField('wsPort', 'number', { filter: filterType.NUMBER });
+		this.addField('state', 'number', { filter: filterType.NUMBER });
+		this.addField('os', 'string', { filter: filterType.TEXT });
+		this.addField('version', 'string', { filter: filterType.TEXT });
+		this.addField('clock', 'number', { filter: filterType.NUMBER });
+		this.addField(
+			'broadhash',
+			'string',
+			{ filter: filterType.TEXT },
+			stringToByte
+		);
+		this.addField('height', 'number', { filter: filterType.NUMBER });
+
+		this.SQLs = {
+			select: this.adapter.loadSQLFile('peers/get.sql'),
+			create: this.adapter.loadSQLFile('peers/create.sql'),
+			update: this.adapter.loadSQLFile('peers/update.sql'),
+			updateOne: this.adapter.loadSQLFile('peers/update_one.sql'),
+			isPersisted: this.adapter.loadSQLFile('peers/is_persisted.sql'),
+		};
+	}
+
+	/**
+	 * Get one peer
+	 *
+	 * @param {filters.Peer|filters.Peer[]} [filters = {}]
```

Comment:
We are mostly using `Array.<Peer>` format in our jsdocs.

Reply:
In accounts it's using the same notation instead of Array.<Account> should we split this to a different issue?